main.go

```diff
@@ -81,31 +81,43 @@ Generate an identity seed and launch a gateway:
 			Name:    "seed",
 			Value:   "",
 			EnvVars: []string{"RAINBOW_SEED"},
-			Usage:   "Seed to derive peerID from. Generate with gen-seed. Needs --seed-index. Best to use $CREDENTIALS_DIRECTORY/seed or $RAINBOW_DATADIR/seed.",
+			Usage:   "Seed to derive peerID from. Generate with gen-seed. Needs --seed-index. Best to use $CREDENTIALS_DIRECTORY/seed or $RAINBOW_DATADIR/seed",
 		},
 		&cli.IntFlag{
 			Name:    "seed-index",
 			Value:   -1,
 			EnvVars: []string{"RAINBOW_SEED_INDEX"},
 			Usage:   "Index to derivate the peerID (needs --seed)",
 		},
-		&cli.StringFlag{
+		&cli.BoolFlag{
+			Name:    "seeds-peering",
+			Value:   false,
+			EnvVars: []string{"RAINBOW_SEEDS_PEERING"},
+			Usage:   "Automatic peering with peers with the same seed (requires --seed and --seed-index). Automatically enables --dht-routing and --dht-shared-host",
+		},
+		&cli.UintFlag{
+			Name:    "seeds-peering-max-index",
+			Value:   100,
+			EnvVars: []string{"RAINBOW_SEEDS_PEERING_MAX_INDEX"},
```

Comment:
:thought_balloon: I realized that since we only have one seed, we should use singular?

```suggestion
			Name:    "seed-peering",
			Value:   false,
			EnvVars: []string{"RAINBOW_SEED_PEERING"},
			Usage:   "Automatic peering with peers with the same seed (requires --seed and --seed-index). Automatically enables --dht-routing and --dht-shared-host",
		},
		&cli.UintFlag{
			Name:    "seed-peering-max-index",
			Value:   100,
			EnvVars: []string{"RAINBOW_SEED_PEERING_MAX_INDEX"},
```